modulemd/v2/modulemd-v2-docs.xml

```diff
@@ -60,4 +63,4 @@
         <xi:include href="xml/test-utils.xml"/>
     </chapter>
   </reference>
-</book>
+</book>
```

Comment:
Quite funny how we keep switching between trailing newline and not...
I believe your IDE might not like them, while mine keeps re-adding them.